.travis.yml

```diff
@@ -1,4 +1,3 @@
 sudo: false
-install: git clone --depth 1 https://github.com/sstephenson/bats.git
-script: PATH="./bats/bin:$PATH" test/run
```

Comment:
OK time for me to sleep (almost 5am here!). Thanks for reviewing and maintaining this awesome project! I am stoked to be able to contribute. I've been using it for years.